### PR TITLE
37: Add download section for the latest 3.7 release

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
This patch updates the downloads page of the website to mention the latest release and link to the download artifacts

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
so far these links don't exist and i haven't verified any of them, since the release is not done.

Not sure what the actual process is - whether we:
1) pre-approve, and I change the links & merge once finished releasing
2) wait for the release, update this, and then get an approval and merge

I would assume 1) is more flexible and preferable, since it allows me to update the website right after the release (avoiding any inconsistency). But I assume also we have time between the actual release and the announcement.

File: downloads.html
Comment (stanislavkozlovski) [inline downloads.html]: This will change to the date of the **release** announcement

File: downloads.html
Comment (stanislavkozlovski) [inline downloads.html]: this needs to get updated 100%, since I don't have the image hash yet

File: downloads.html
Comment (divijvaidya) [inline downloads.html]: typo: this should point to 370 release blog

**Comment by divijvaidya** [comment]
> But I assume also we have time between the actual release and the announcement.

Yes, we can publish this PR *after* uploading artifacts to maven etc.

File: downloads.html
Comment (stanislavkozlovski) [inline downloads.html]:
```suggestion
        For more information, please read our <a href="https://kafka.apache.org/blog#apache_kafka_370_release_announcement" target="_blank">blog post</a>
```

**Review by divijvaidya — APPROVED** [review]
Conditional approval assuming we would update the date and docker image link after uploading the artifacts. Please feel free to merge after 37 artifacts are uploaded and you have updated the date/dicker image link, without waiting for another approval.

**Comment by stanislavkozlovski** [comment]
updated the docker image and the release date, to today

**Pull request merged**
